algorithms/src/polycommit/kzg10/data_structures.rs

```diff
@@ -232,19 +154,7 @@ impl<E: PairingEngine> ToBytes for UniversalParams<E> {
     }
 }
 
-impl<E: PairingEngine> UniversalParams<E> {
-    pub fn max_degree(&self) -> usize {
-        self.powers.read().degree() - 1
-    }
-
-    pub fn supported_degree_bounds(&self) -> &[usize] {
-        &self.supported_degree_bounds
-    }
-
-    pub fn increase_degree(&self, degree: usize) -> Result<()> {
-        self.download_up_to(degree)
-    }
-}
+impl<E: PairingEngine> UniversalParams<E> {}
```

Comment:
Is this still needed?

Reply:
No, removed in [`1e21f75` (#1245)](https://github.com/AleoHQ/snarkVM/pull/1245/commits/1e21f7525c2d54d68146fada724bc611092b75f7)